utils/gear-runtime-test-cli/src/command.rs

```diff
@@ -69,20 +72,63 @@ impl GearRuntimeTestCmd {
         let total_failed = AtomicUsize::new(0);
 
         println!("Total fixtures: {}", total_fixtures);
-        tests.par_iter().for_each(|test| {
-            test.fixtures.par_iter().for_each(|fixture| {
-                new_test_ext().execute_with(|| {
-                    gear_common::reset_storage();
-                    let output = run_fixture(test, fixture);
-                    MailboxOf::<Runtime>::remove_all();
-
-                    println!("Fixture {}: {}", fixture.title.bold(), output);
-                    if !output.contains("Ok") && !output.contains("Skip") {
-                        total_failed.fetch_add(1, Ordering::SeqCst);
-                    }
-                });
-            });
-        });
+        let (executions, times) = tests
+            .par_iter()
+            .map(|test| {
+                let (fixtures, times) = test
+                    .fixtures
+                    .par_iter()
+                    .map(|fixture| {
+                        new_test_ext().execute_with(|| {
+                            gear_common::reset_storage();
```

Comment:
btw, why do we need storage reset when we run with new test ext?

Reply:
If I'm not mistaken to reset old data of the previous executions since it is written with `spi_io` yet

Reply:
Strange thing, due to the fact of clear in-memory env in new test ext.
Ok, just run into this and decided to ask. Not related to this PR